helpers/docker_wrapper.sh

```
@@ -0,0 +1,114 @@
+    DOCKER_COMPOSE_CMD="docker compose"
+  fi
+  return 0
+}
+
+_detect_docker_mode
+
+export DOCKER_CMD
+export DOCKER_COMPOSE_CMD
+
+run_docker() { # run_docker <args...>
+  eval "$DOCKER_CMD" "$@"
```

Shellcheck (0.11.0)

[warning] eval negates the benefit of arrays. Drop eval to preserve whitespace/symbols (or eval as string).

(SC2294)